modules/vba_documents/lib/tasks/one_time_status_update_20200803.rake

```diff
@@ -0,0 +1,2210 @@
+      740f9185-9efe-4017-a1ed-e915f9c964ea
+      75162bac-ff63-404e-989a-ddc49cb3c952
+      767775588be4bec16686c0c0
+      7bd9b0be-1426-4742-807c-b2f77c96b051
+      7feed306-d7c0-44c1-88b8-02f3dd250189
+      8175c6c5de544785ea1cb44f
+      837a5be4-00a8-4ad9-89e7-a6aa00105c00
+      848c5bfd-a109-420e-b83e-ac8044bd7926
+      880b0d0f-a7aa-4660-9d99-44820e75df8e
+      8a89df86-5fe9-4dfc-8afa-e570fa42447e
+      8a9e7643-0757-475d-8ec3-1c6399101a63
+      8c1855327c91e4bf147d8edd
+      8d21fd34-c2d8-4462-bd2a-d7978f19a919
+      8e65c4bd-6ec5-4d38-ad8d-931762361842
+      92802e7e-94a9-40d9-b1ba-da7a12c66995
+      936d915e-0644-4922-a797-dbacb1a226e3
+      94e2539ef149efc506cb795c
+      975b83c8eca84eb81e40f0eb
+      978bdfed-2761-4d49-b482-53f900214ea7
+      986d6a28818898dd57f12818
+      9df9226fe362ca2be75fc520
+      9f5e39d371abef752a5969f1
+      a035f0ed-1c46-4621-b16d-cc9d6d6d47bf
+      a16a4959-58c0-45a2-ace7-4c067a450eff
+      a3b3baca-ce74-4e26-b423-e504c80cf40c
+      a73431a6-598c-4cae-8c86-a76c0fee4735
+      a84c036b6057bdc9b164a9f4
+      ab8b23f4c2b7e7d473944195
+      ad6b90a1-00e1-4ab0-8393-917b65052e7c
+      773ce202-0aab-46e1-bbb5-4ef9c0b17416
+      7810d4b3-2873-4065-a739-f341e1330b18
+      7882af2ee2c989c885842c36
+      78c496b4-06b7-4685-bd0e-95db7f406bab
+      78d59d67-98b5-4cdd-86ac-d62bb8f93497
+      799a32ac-2a47-4338-b7e8-42c6cbbc5a18
+      7a2cff32-0b46-4db1-a8b7-f4098e24f102
+      7a6b7929-ccdd-4aa2-b7c0-e77147d2923f
+      7a8f579d-3a67-41b7-9ac5-18f34c634ae6
+      7ace216f-3cdd-40ad-bd8d-6715a389440f
+      7af19b8dbb6afc7439c93080
+      7b0f8a87-b905-4785-8432-77c446a998bd
+      7b6aef64-d47e-446e-b2f9-6a738e55e176
+      7b9025ec-42af-4d11-93a1-05c4b6162ca9
+      7bad8b01-b24d-4c88-b4d1-74ad8318ffca
+      7c714911-b1f1-477f-94d6-3101dc62e3c2
+      7c93e682-590d-409b-87b9-0956b13dec01
+      7cbf4aa5f0a90c2e406b280b
+      7cd348d7-1947-43d6-a61c-06f3e78f301b
+      7cdb63a7fb0f1cb776e967fe
+      7d0f5971-b046-47d7-a1a2-70d32507afd9
+      7dc69339-dd5e-4326-af5e-ff18562920a9
+      7dfcba9f-5f8f-4500-b149-631e0e12f773
+      7e424e0e-0424-475e-81c8-e5148caba5ce
+      7e4708777660f9c0b752ebf4
+      7e65d5bb-b055-4ab7-816f-2637e44d3d2d
+      7f6b18ff-f2a1-48ab-aea4-36a865151bc8
+      8040c0c0-4973-4316-94cd-bf2ad4569571
+      805bfb90-6257-465d-bbcc-cfb2df2cdf94
+      80bbacc6-2dbf-419f-a14e-f33373bed63d
+      80c3b49c-26a5-4da6-997f-926380eb90ff
+      80dd1b3e-b834-4405-84de-7010d6492f95
+      80e37956-f4ae-4c08-83e3-e9dfd7509a57
+      80ecfbd4-d3eb-4bf5-b8f3-cc0596b81cfa
+      80fcebbb-8602-4274-8280-b71a28bb4045
+      4bc95949-3abe-4270-a032-e48f11345cc0
+      4c005633-0d87-4a46-b85f-da22f5fca862
+      4c461012-5355-46d0-9077-88a14979fb59
+      4c78d2b1-96b7-44f5-b383-b788064fa5cf
+      4cb4132b-6cac-4976-a424-1902c50ead1a
+      4ce8cc8b-928e-45ba-b2c4-c16ba58d3af4
+      4d839061-de4c-43c8-aaaf-12a4515172d4
+      4db6305811e2700f66ee09fe
```

Comment:
These don't belong to us at all, just remove anything that doesn't look like a standard UUID and we can turn these over to the va.gov side.

Reply:
when this rake task runs, it will print out a list of bad uuids (as well as other errors). that's why i left them in

Reply:
it's also a nice record of what they gave us